pypit/armasters.py

```diff
@@ -120,51 +124,50 @@ def save_masters(slf, det, setup):
 
     transpose = bool(settings.argflag['trace']['dispersion']['direction'])
 
-    # MasterFrame directory
-    mdir = settings.argflag['run']['directory']['master']+'_'+settings.argflag['run']['spectrograph']
     # Bias
     if 'bias'+settings.argflag['reduce']['masters']['setup'] not in settings.argflag['reduce']['masters']['loaded']:
         if not isinstance(slf._msbias[det-1], (basestring)):
             arsave.save_master(slf, slf._msbias[det-1],
-                               filename=master_name(mdir, 'bias', setup),
+                               filename=master_name('bias', setup),
                                frametype='bias')
     # Bad Pixel
     if 'badpix'+settings.argflag['reduce']['masters']['setup'] not in settings.argflag['reduce']['masters']['loaded']:
         arsave.save_master(slf, slf._bpix[det-1],
-                               filename=master_name(mdir, 'badpix', setup),
+                               filename=master_name('badpix', setup),
                                frametype='badpix')
     # Trace
     if 'trace'+settings.argflag['reduce']['masters']['setup'] not in settings.argflag['reduce']['masters']['loaded']:
         extensions = [slf._lordloc[det-1], slf._rordloc[det-1],
                       slf._pixcen[det-1], slf._pixwid[det-1],
                       slf._lordpix[det-1], slf._rordpix[det-1]]
+        names = ['LeftEdges', 'RightEdges', 'Pixcen', 'Pixwid', 'LordPix', 'RordtPix']
```

Comment:
I should probably clarify what these arrays are...

```
slf._lordloc[det-1] = LeftEdges (detector coordinates)
slf._rordloc[det-1] = RightEdges (detector coordinates)
slf._pixcen[det-1] = SlitCentre
slf._pixwid[det-1] = SlitLength (in pixels)
slf._lordpix[det-1] = LeftEdges (pixel coordinates)
slf._rordpix[det-1] = RightEdges (pixel coordinates)
```

I think we should raise an issue to think more carefully about whether or not all of these arrays are needed. I originally distinguished between these for high precision analyses, where the gaps between pixels affected the wavelength calibration. This distinction is most certainly not for needed for longslit work. I'll raise an issue now.